web/libs/datamanager/src/components/DataManager/Toolbar/ActionsButton.scss

```
@@ -0,0 +1,184 @@
+.actionButton {
+  --compact-height: 32px;
+  height: var(--compact-height);
+  padding: 0 13px;
+  display: flex;
+  align-items: center;
+
+  &__titleContainer {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    justify-content: space-between;
+  }
+
+  &_isTitle {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.5px;
+    color: var(--grape_500);
+    cursor: initial;
+    padding: 16px 12px 8px;
+    height: var(--compact-height);
+
+    .actionButton {
+      &__titleContainer:hover {
+        background-color: transparent;
+      }
+
+      &__title {
+        font-size: 12px;
+      }
+    }
+
+    &:hover {
+      background-color: transparent !important;
+      color: var(--grape_500) !important;
+    }
+  }
+
+  &_isSeparator {
+    cursor: initial;
+    height: 1px;
+    background-color: rgba(137, 128, 152, 0.12) !important;
+    padding: 0;
+
+    .actionButton__titleContainer {
+      display: none;
+    }
+
+    &.menu__item {
+      height: 1px;
+    }
+
+    &:hover {
+      background-color: transparent;
+    }
+  }
+
+  &_hasSeperator {
+    border-top: 1px solid rgba(137, 128, 152, 0.12);
+  }
+
+  &__title {
+    flex-grow: 1;
+    font-size: 16px;
+  }
+
+  &_hasSubMenu {
+    color: var(--sand_600);
+
+    &:hover {
+      color: #000;
+      background-color: rgba(0, 0, 0, 0.04);
+    }
+  }
+
+  &_danger {
+    color: #d00;
+  }
+
+  &_disabled {
+    background-color: rgba(0, 0, 0, 0.04);
+    color: rgba(0, 0, 0, 0.25);
+    cursor: initial;
+
+    &:hover {
+      background-color: rgba(0, 0, 0, 0.04) !important;
+      color: rgba(0, 0, 0, 0.25) !important;
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
      background-color: rgba(var(--black), 0.04) !important;
      color: rgba(var(--black), 0.25) !important;
```